src/Cli.Tests/Cli.Tests.csproj

```diff
@@ -14,7 +14,10 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="coverlet.msbuild" />
+        <PackageReference Include="coverlet.msbuild">
+          <PrivateAssets>all</PrivateAssets>
```

Comment:
why was this change required?

Reply:
this change seems to mirror that sub-properties of the other package reference: `coverlet.collector` 
The intent is to limit the content sent to the build output folder from this dependency: https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets